docs/Users_Guide/overview.rst

```diff
@@ -0,0 +1,277 @@
+private sector, and operational modeling and prediction centers.
+
+
+METviewer Requirements
+----------------------
+
+METviewer requires installation of Java, Apache / Tomcat, MySQL, and R
+statistics.
+
+.. figure:: figure/MV_flowchart.png
+	    
+	    Figure 1.1 METviewer flowchart, showing types of input data,
+	    output types, and options for use. 
+
+
+
+.. _userspec:
+
+METviewer User Specifications
+-----------------------------
+
+The top menu allows the user to select a database and type for the plot. 
+
+The first section is where the dependable variable (Y-axis) , statistic of
+interest, and series variables of interest are selected. The Y2 axis can be
+used to plot a Base Rate.
+
+The fixed variables section controls the constraints for the plot (what
+period of data is used, what thresholds to include, interpolation method, as
+well as domains). Fixed values are for stratifications. For example, select
+one or more thresholds to be aggregated, or one or more initialization
+times, or a date range based on valid times or cycle time.
+
+Each of the above selections can have one or multiple items. To add
+variables, series, or fixed values, select the button with the '+' directly
+below the desired item. Then additional selections will appear in a new
+dropdown menu. To delete these extras, select the trash icon to the left of
+the item. 
+
+The independent variable section is where the X-axis is configured. As an
+example, forecast lead hour often serves as the x-axis for a series plot.
+
+This section is reserved for line labels and formatting for the individual
+series confidence intervals: Normal Bootstrap Standard error (median or
+mean) – configurable. 
+
+Summary – scores computed per each combination of fixed values and
+independent variable then mean or median taken. 
+
+Aggregate statistics – accumulates SL1L2 lines or CTC counts prior to
+calculating the statistics.
+
+Confidence intervals - METviewer supports two statistical derivation
+methods... (1) plot the mean of the daily stats with confidence intervals
+defined by the "standard error"... that uses the standard deviation to add
+confidence intervals and assumes normality (2) plot the aggregated
+statistics with confidence intervals defined by bootstrapping that aggregation.
+
+Pairwise Differencing – computes the difference in statistics for two models
+on a pairwise basis (same forecast hour, same initialization, same region,
+etc…).  Confidence intervals may be applied to determine statistical
+significance.
+
+Event Equalization – performs the filtering on two datasets to produce a
+“homogeneous” intersection between the samples (same forecast hour, same
+initialization, same region, etc…).  Use of this method is recommended when
+trying to compare the skill of one or more predictions.
+To test, I tried method (1) and it works fine for some of the categorical
+stats, like BASER and FMEAN. However, applying it to Gilbert Skill Score  (GSS) results in this error:
```

Comment:
> To test, I tried method (1) and it works fine for some of the categorical
stats, like BASER and FMEAN. However, applying it to Gilbert Skill Score  (GSS) results in this error:

Hi Lindsay.  This line of text refers to "this error", with a colon after it as if an error were going to be beneath it, but there is no error.  Do you have an error to add here?  If not, could you please reword it to say "results in an error." or something like that?  Please do a "git pull" before starting on any changes, as I know Lisa committed some changes to this file.  Thanks!